dev/requirements.txt

```diff
@@ -60,6 +60,9 @@ mypy-protobuf==3.3.0
 googleapis-common-protos-stubs==2.2.0
 grpc-stubs==1.24.11
 
+# Debug for Spark and Spark Connect
+graphviz==0.20.3
```

Comment:
This would have to be installed in `dev/infra/Dockerfile`. Could be done separately in another PR.

Reply:
and would be great if we can mention this as an optional dependency ay `python/docs/source/getting_started/install.rst`.